CGPs/cgp-0146.md

```diff
@@ -0,0 +1,39 @@
+---
+cgp: 146
+title: MU07 - ChainlinkRelayer Oracle Upgrade
+date-created: 2024-08-26
+author: "Philip Rätsch <philip.raetsch@mentolabs.xyz>"
+status: DRAFT
+discussions-to: https://forum.celo.org/t/mu07-chainlink-price-feed-integration/8850
+governance-proposal-id: TODO
+date-executed: TBD
```

Comment:
Please leave this fields empty otherwise Rosy's validation tool is not gonna pass.